setup.sh

```diff
@@ -250,7 +250,7 @@ function synchronize_files()
   cmd_output_manager "cp $app_name $binpath" "$verbose"
   ASSERT_IF_NOT_EQ_ZERO "The command 'cp $app_name $binpath' failed" "$?"
 
-  sed -i -e "s,##KW_INSTALL_PREFIX_TOKEN##,$PREFIX/,g" "$binpath/$app_name"
+  sed -i -e "s,##KW_INSTALL_PREFIX_TOKEN##,$PREFIX,g" "$binpath/$app_name"
```

Comment:
was this intentional? i dont see how it relates to the rest of the commit

Reply:
This was causing KW_LIB_DIR to have two slashes in it's name, making the code in `include_manager` not function properly